### PR TITLE
Handles permission denied exceptions from Bugzilla

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: ci/redmine_bugzilla.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 from bugzilla.rhbugzilla import RHBugzilla
 from redmine import Redmine
 import urllib3.contrib.pyopenssl
+import xmlrpclib
```
Comment (bmbouter) [inline ci/redmine_bugzilla.py]: This dep probably also needs to be installed [here](https://github.com/pulp/pulp_packaging/blob/master/ci/jobs/redmine.yaml#L40) right?

**Comment by bmbouter** [comment]
Can some logging be added before having it `continue` after catching the code 102?

File: ci/redmine_bugzilla.py (hunk 1 above)
Comment (dkliban) [inline ci/redmine_bugzilla.py]: no ... it gets installed with the bugzilla libraries

Comment (bmbouter) [inline ci/redmine_bugzilla.py]: You're right, I just tested this myself

**Pull request merged**